### ISSUE TITLE
not all ELEMENTX-ELEMENTY bonds are present in BOND_ORDERS lookup dictionary

### ISSUE THREAD

**Issue by EyalRozenberg1** [a-r-j/graphein#82]
`BOND_ORDERS` includes only part of all optional bonds. e.g S-S is not present but indeed optional. 
Thus, at line 223 in `graphein.protein.edges.atomic.identify_bond_type_from_mapping` the following `allowable_order = BOND_ORDERS[query]` fails.

I assume the solution should be in  `graphein.protein.edges.atomic.add_bond_order` (line 188), as follows:
```
            try:
                identify_bond_type_from_mapping(G, u, v, a, query)
            except:
                query = f"{atom_b}-{atom_a}"
                try:
                    identify_bond_type_from_mapping(G, u, v, a, query)
                except:
                    G.edges[u, v]["kind"].add("SINGLE")
```

**Comment by a-r-j** [comment]
Hi @EyalRozenberg1 Thanks for the bug reports - I appreciate it! I've added fixes to this PR (#83) - would you mind scoping them out when you've got a chance? I'll merge & reupload to PyPI ASAP :)

#80 Should be fixed in the repo already, but I don't think it's reflected in the version on PyPI

**Comment by EyalRozenberg1** [comment]
Looks good @a-r-j . Keep up the good work! 
Thanks
